inc/namespace.php

```diff
@@ -80,6 +80,20 @@ function merge_config_settings( array $config, array $overrides ) : array {
 			// Merge module settings together.
 			case 'modules':
 				foreach ( $value as $module => $settings ) {
+
+					// Check settings syntax is valid.
+					if ( ! is_array( $settings ) && ! is_bool( $settings ) ) {
```

Comment:
@roborourke @dashaluna Shouldn't this be a `||` instead of `&&`?

Reply:
No, if `$settings` was a boolean then the `! is_array()` check would return true and the if statement would be executed. It has to be not an array _and_ not a boolean to trigger the warning.